### DISCUSSION TITLE
Support React Native on @xstate/inspect

### DISCUSSION THREAD

**Discussion by kieran-osgood** [statelyai/xstate#3460] [Ideas]
@xstate/inspect doesn't support react-native environments, attempted to use `createWindowReceiver` too but in react-native you don't have access to the global window object. 

The iFrame approach also wouldn't work within react-native, so assume it'd be best to just default to the opening new browser window

![image](https://user-images.githubusercontent.com/17687552/179472654-5bf20138-5f45-4f7f-8ab8-7c48faef1a50.png)

(Filed at request of @davidkpiano: https://discord.com/channels/795785288994652170/801135010265825300/998262103519547402)

I'm happy to help out as a tester or anything :)

**Comment by cpetzel** [comment]
I built a React Native Flipper plugin that loads the xstate inspector. There are a few minor issues with it, but it is mostly useable. Feel free to use it as a baseline for the official implementation! 

https://github.com/cpetzel/flipper-plugin-xstate
and
https://github.com/cpetzel/react-native-flipper-xstate
